factory_girl_rails.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
     factory_girl and rails 3 or newer (currently just automatic factory definition
     loading)}
 
-  s.files         = Dir['**/*'].keep_if { |file| File.file?(file) }
+  s.files         = `git ls-files`.split("\n").reject { |f| f.match(%r{^(spec|features)/}) }
```

Comment:
FYI, bundler/rubygems will raise a warning (or maybe not work at all? I forget which version this started in) with the usage of running subcommands like this.

I think that ignoring spec and features for gem packaging makes sense, but can you update this to remove the `git ls-files...` addition?

Reply:
Yeah, agreed that it's popular ... I could have sworn that I saw something advising against using it. Maybe I'm thinking of something else?!

Reply:
I found the related issue at bundler/bundler#5810. This issue describes installing a gem from source in an environment without git is impossible. bundler core-developer replied in following comment.
https://github.com/bundler/bundler/issues/5810#issuecomment-310480516